internal/manifest/os.go

```diff
@@ -95,6 +95,7 @@ type OSCustomizations struct {
 	WAAgentConfig       *osbuild.WAAgentConfStageOptions
 	UdevRules           *osbuild.UdevRulesStageOptions
 	LeapSecTZ           *string
+	FactAPIType         *string
```

Comment:
nitpick: IMO this could be a plain `string`, because an empty string is probably not a valid value and we can use it to check if it is set. This would make it a bit "easier" to use and not to have to do the dance with pointers...

Reply:
Thought about it and thought it might be valid to have an empty string and just the key, but maybe not for our purposes.  Will change.

Reply:
Fixed.